apollo-portal/src/main/resources/static/open/manage.html

```
@@ -79,11 +80,19 @@ <h5>{{'Open.Manage.CreateThirdApp' | translate }}
                                 </td>
-                                <td style="width: 10%">
+                                <td style="width: 10%" >
+                                    <div ng-if="consumer.allowManageUsers">
+                                        <b>{{'Open.Manage.Consumer.AllowManageUsers.Yes' | translate}}</b>
+                                    </div>
+                                    <div ng-if="!consumer.allowManageUsers">
+                                        {{'Open.Manage.Consumer.AllowManageUsers.No' | translate}}
+                                    </div>
+                                </td>
+                                <td style="width: 8%">
                                     {{ consumer.rateLimit && consumer.rateLimit > 0 ? consumer.rateLimit : 'Open.Manage.Consumer.RateLimitValue.Display' | translate }}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)